src/EA.Weee.Web/Areas/Admin/Views/AddAatf/SearchResults.cshtml

```diff
@@ -36,7 +36,7 @@
                     m => m.SelectedOrganisationId,
                     Model.Results,
                     r => r.OrganisationId,
-                    r => r.Name,
+                    r => r.Name  +" " + r.AddressString,
```

Comment:
Rather than doing this can with have NameWithAddress property on the search result, then we can test that property